### PR TITLE
Updates Containerized Docker Application Lifecycle - ebook content to .NET 6.0

### PR DESCRIPTION
**Pull request by sughosneo**
This PR updates the `Containerized Docker Application Lifecycle with Microsoft Platform and Tools` ebook content to .NET 6.0

### CODE REVIEW

File: docs/architecture/containerized-lifecycle/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md [hunk 1]
```diff
@@ -16,33 +16,33 @@ AKS main features are:
...
+Verify that you've selected ASP.NET Core 6.0 as the framework. .NET 6 is included in the latest release of Visual Studio 2022 and is automatically installed and configured for you when you install Visual Studio.
 
 ![Visual Studio dialog for selecting the type of an ASP.NET Core Web Application with API option selected.](media/build-aspnet-core-applications-linux-containers-aks-kubernetes/create-web-api-application.png)
 
-**Figure 4-37**. Selecting ASP.NET CORE 5.0 and Web API project type
+**Figure 4-37**. Selecting ASP.NET CORE 6.0 and Web API project type
 
-Notice Docker support is not enabled now. You'll do that in the next step after the project creation.
+Notice Docker support is not enabled now. You'll do that in the next step after the project creation. You'll also notice that by default controller option is checked. You can uncheck that if you want to [Create a minimal web API with ASP.NET Core](https://docs.microsoft.com/aspnet/core/tutorials/min-web-api?view=aspnetcore-6.0&tabs=visual-studio).
```
Comment (IEvangelist) [inline docs/architecture/containerized-lifecycle/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md:45]:
```suggestion
Notice Docker support is not enabled now. You'll do that in the next step after the project creation. You'll also notice that by default controller option is checked. You can uncheck that if you want to [Create a minimal web API with ASP.NET Core](/aspnet/core/tutorials/min-web-api?view=aspnetcore-6.0&tabs=visual-studio).
```

**Review by IEvangelist — APPROVED** [review]
Great job on this @sughosneo, I just have one nit - otherwise we'll :shipit:

**Comment by nishanil** [comment]
@sughosneo As discussed, there are a few pages that has references to VS 2019 that needs to be fixed. Can you update those changes too within this PR, please?

**Comment by sughosneo** [comment]
> @sughosneo As discussed, there are a few pages that has references to VS 2019 that needs to be fixed. Can you update those changes too within this PR, please?

@nishanil  - I have updated the content. Could you please review again?

**Comment by nishanil** [comment]
@sughosneo The page "visual-studio-tools-for-docker.md" has a few references to VS 2017. Can you also update them, please?

File: docs/architecture/containerized-lifecycle/docker-terminology.md [hunk 2]
```diff
@@ -29,17 +29,15 @@ docker build
...
-
-**Docker Enterprise Edition (EE)**: An enterprise-scale version of Docker tools for Linux and Windows development.
+**Docker Desktop**: Development tools for Windows and macOS for building, running, and testing containers locally. Docker Desktop for Windows provides development environments for both Linux and Windows Containers. The Linux Docker host on Windows is based on a [Hyper-V](https://www.microsoft.com/cloud-platform/server-virtualization) virtual machine. The host for Windows Containers is directly based on Windows. Docker Desktop for Mac is based on the Apple Hypervisor framework and the [xhyve hypervisor](https://github.com/mist64/xhyve), which provides a Linux Docker host virtual machine on Mac OS X. Docker Desktop for Windows and for Mac replaces Docker Toolbox, which was based on Oracle VirtualBox.
```
Comment (nishanil) [inline docs/architecture/containerized-lifecycle/docker-terminology.md:40]:
```suggestion
**Docker Desktop**: Development tools for Windows and macOS for building, running, and testing containers locally. Docker Desktop for Windows provides development environments for both Linux and Windows Containers. The Linux Docker host on Windows is based on a [Hyper-V](https://www.microsoft.com/cloud-platform/server-virtualization) virtual machine. The host for Windows Containers is directly based on Windows. Docker Desktop for Mac is based on the Apple Hypervisor framework and the [xhyve hypervisor](https://github.com/mist64/xhyve), which provides a Linux Docker host virtual machine on macOS. Docker Desktop for Windows and for Mac replaces Docker Toolbox, which was based on Oracle VirtualBox.
```

**Review by nishanil — APPROVED** [review]

**Pull request merged**